packages/legacy/app/container-imp.tsx

```diff
@@ -20,6 +21,30 @@ export class AppContainer implements Container {
     // Here you can register any component to override components in core package
     // Example: Replacing button in core with custom button
     // this.container.registerInstance(TOKENS.COMP_BUTTON, Button)
+
+    //Customizing Terms screen custom header
+    /*
+    this.container.registerInstance(TOKENS.OBJECT_ONBOARDING_CONFIG, {
+      ...DefaultScreenOptionsDictionary,
+      [Screens.Terms]: {
+        ...DefaultScreenOptionsDictionary[Screens.Terms],
+        headerShown: false,
+      },
+    })
+    this.container.registerInstance(TOKENS.OBJECT_LAYOUT_CONFIG, {
+      ...DefaultScreenLayoutOptions,
+      [Screens.Terms]: {
+        ...DefaultScreenLayoutOptions[Screens.Terms],
+        customEdges: ['bottom'],
+        safeArea: true,
+        Header: () => (
+          <View style={{ backgroundColor: 'red', height: 129, justifyContent: 'center', alignItems: 'center' }}>
+            <Text style={{ color: 'white' }}>Custom Header</Text>
+          </View>
+        ),
+      },
+    })
+      */
```

Comment:
?

Reply:
This is an example, not sure if its a good idea to leave it here or maybe document it somewhere or remove it and developers figure out

Reply:
I'm a little torn on this as well, what about moving this into the wrapper itself? I think it would be useful to have an example of this for someone who might not fully understand how to implement this on a screen in the future. It would also be great to have a brief explanation as to what this is an example of and maybe provide a little more context to the wrapper itself a